tests/test_parallel_loading.py

```
@@ -0,0 +1,115 @@
+    assert req.status_code == 200, req.content
+    index_info = json.loads(req.content)
+    
+    print(f"Small index has {index_info['segments']} segments before restart")
+    
+    # Restart - should use sequential loading
+    server.restart()
+    server.wait_for_ready(index_name, timeout=10.0)
+    
+    # Verify functionality
+    req = client.post(f'/{index_name}/_search', json={
+        'query': [100, 200, 300]
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)